.github/workflows/ci.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - master
+      - 0.*
```

Comment:
That's a good idea.  I guess Travis does this automatically.  Our release branches are always have names like `A.B.x`, so perhaps a `"*.x"` would be better?